### ISSUE TITLE
Clarify status of the project

### ISSUE THREAD

**Issue by thomasbertet** [apollographql/federation-jvm#9]
Hi there, 

First thank you for trying to filling the gap between graphql-java & the GraphQL Federation spec. Really appreciate it.

As Federation is going to be more and more popular and some people will want to use it (like me) in a Java project, it would be great to know the current status of this project/lib.

As it seems, the project looks pretty early and maybe not suitable for production use. 

It would be interesting to know what you (the creators/maintainers) think of the current version of the lib & what's missing  - in short a kind of a roadmap.

Also, can we expect this project to go along ? Are you willing to continue its development further ?

Anyway, thank you for this work and thank you for any insight you can give us so we can plan -or not- any Federated project :)

**Comment by bbortt** [comment]
It is clearly not usable at the moment.. The example's not even working :(

**Comment by mcohen75** [comment]
The project does work when #3 is merged. This is the only issue that prevents it from working. The structure of the project does need some attention as it currently pulls in a number of unnecessary dependencies.

**Comment by bbortt** [comment]
+1

**Comment by pcarrier** [comment]
We're in the process of dogfooding this with the Engine service. It is intended to be functional and maintained, any feedback is certainly welcome.

@mcohen75 not sure about unnecessary dependencies; could you detail your observations?

```
% mvn -pl graphql-java-support dependency:tree -Dscope=runtime
[INFO] Scanning for projects...
[INFO]
[INFO] ----< com.apollographql.federation:federation-graphql-java-support >----
[INFO] Building federation-graphql-java-support 0.1.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ federation-graphql-java-support ---
[INFO] com.apollographql.federation:federation-graphql-java-support:jar:0.1.0
[INFO] \- com.graphql-java:graphql-java:jar:12.0:compile
[INFO]    +- org.antlr:antlr4-runtime:jar:4.7.2:compile
[INFO]    +- org.slf4j:slf4j-api:jar:1.7.25:compile
[INFO]    +- com.graphql-java:java-dataloader:jar:2.1.1:compile
[INFO]    \- org.reactivestreams:reactive-streams:jar:1.0.2:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.134 s
[INFO] Finished at: 2019-07-07T18:33:08-07:00
[INFO] ------------------------------------------------------------------------
```

**Comment by thomasbertet** [comment]
Thanks @pcarrier. It would be great if maybe you could update the README with any informations you currently have, so others don't have to dig into this issue.

**Comment by pcarrier** [comment]
@thomasbertet Which issue are you referring to? AFAICT we don't have any identified bug in `federation-jvm` at the moment.

**Comment by acatxnamedvirtue** [comment]
I think @thomasbertet is referring to the current status of the project and a roadmap. The README could benefit from some updates there, and also maybe a "How to get started using federation-jvm", if the project is at the point where external parties can use it in their graphql-java projects.

**Comment by thomasbertet** [comment]
Yeah in fact @pcarrier I did not point out any issue with the code itself, but @mcohen75 did  :)

**Comment by pcarrier** [comment]
@acatxnamedvirtue @thomasbertet merged some additions to README.md ; feedback and contributions more than welcome!

**Comment by DeanJain** [comment]
Pls provide update on your roadmap and 1.0.0 release ?

**Comment by sachindshinde** [comment]
Sorry for the lack of communication here, we'll have some more information about project status and roadmap by March 9th.

**Comment by mmuth** [comment]
@sachindshinde any updates on this :) ??

**Comment by sachindshinde** [comment]
@mmuth 
Apologies for the delay, the main focus right now is to get the changes out for compatibility with graphql-java v14.

After that I'll need to talk to the relevant people on teams for a roadmap.

**Comment by setchy** [comment]
Any further news / visibility on a roadmap @sachindshinde

**Comment by dariuszkuc** [comment]
Superseded by https://github.com/apollographql/federation-jvm/issues/173
